### ISSUE TITLE
How can I set the defaul icon in the extracted exe using wails build, wails 2.3.1 version

### ISSUE THREAD

**Issue by kishorkumar287** [wailsapp/wails#2360]
### Description

How to remove the wails icon in the extracted EXE.

### To Reproduce

wails build 

### Expected behaviour

Need to have the windows default icon as in the extracted EXE file.

### Screenshots

![image](https://user-images.githubusercontent.com/60978027/217500016-a1e2410a-f940-46da-a4bb-392f234f045f.png)

### System Details

```shell
DEB | Using go webview2loader
Wails CLI v2.3.1
Scanning system - Please wait (this may take a long time)...Done.
# System
OS           | Windows 10 Pro
Version      | 2009 (Build: 22621)
ID           | 22H2
Go Version   | go1.19.2
Platform     | windows
Architecture | amd64
# Wails
Version | v2.3.1
# Dependencies
Dependency | Package Name | Status    | Version
WebView2   | N/A          | Installed | 109.0.1518.78
npm        | N/A          | Installed | 8.19.3
*upx       | N/A          | Available |
*nsis      | N/A          | Available |
* - Optional Dependency
# Diagnosis
Your system is ready for Wails development!
Optional package(s) installation details:
  - upx : Available at https://upx.github.io/
  - nsis : More info at https://wails.io/docs/guides/windows-installer/
 :hearts:   If Wails is useful to you or your company, please consider sponsoring the project:
https://github.com/sponsors/leaanthony
```

**Comment by akuehnis** [comment]
In my application I have two image files:

- /build/appicon.png (see https://wails.io/docs/gettingstarted/firstproject)
- /build/windows/icon.ico

Works perfect for me. Is this what you are looking for?

**Comment by badarsebard** [comment]
It's not working for me. Are there any requirements around the file?

**Comment by marrasen** [comment]
I can't get it to work either. I have an old project that works, I tried upgrading that project to the latest version of wails and rebuild that project, and it works fine. Then I tried to copy the icon from that project to my newer, but that project still won't work. I even tried the other way around, copying my new icon to the old project, the old project got the new icon just fine.

I tried creating an entirely new project, and then the icon works fine as well. I've tried "build -clean" and deleting all build folders and regenerating the icon and converting it to a ico-file using other tools than wails, but to no success.

So this is not a problem with the requirements of the icon. Something else very strange is going on.

**Comment by leaanthony** [comment]
Are you using the same CLI version for bothe old and new projects?

**Comment by marrasen** [comment]
Yes, I upgraded to the latest version, 2.4.1 before performing the tests I mentioned. I had to manually modify my go.mod from 2.4.0 in the newer project, but the go.mod for my older project updated automatically, maybe because I'm using the same Go installation (?)

**Comment by marrasen** [comment]
After a lot of trial and error I managed to get the icon working. If I changed the "name" field in "wails.json" it started working. If I changed it back it stopped working again. If I created a brand new project and entered the same name it stopped working in that project. I cannot find any pattern to why this happens. I didn't use any special characters or exceptionally long name. 

Is the name used for caching the generated icon or something somewhere? I'm curious to try on another computer. I'm on Windows.

(I don't think my customer wants me to provide the name I used that causes this to happen, but I can ask)
